docs/en/stack/ml/nlp/ml-nlp-deploy-models.asciidoc

```diff
@@ -186,7 +186,9 @@ NOTE: Since eland uses APIs to deploy the models, you cannot see the models in
 When you deploy the model, its allocations are distributed across available {ml} 
 nodes. Model allocations are independent units of work for NLP tasks. To 
 influence model performance, you can configure the number of allocations and the 
-number of threads used by each allocation of your deployment.
+number of threads used by each allocation of your deployment. Alternatively, you
+can enable <<nlp-model-adaptive-allocations>> to automatically create and remove
+model allocations based on the current workload of the model.
```

Comment:
To be precise: adaptive allocations is an alternative to setting the number of allocations. It's not an alternative to setting the number of threads. That has to be done regardless.